### ISSUE TITLE
kotlinter 3.13.0 crashes when running with ktlint 0.48.2

### ISSUE THREAD

**Issue by kkocel** [jeremymailen/kotlinter-gradle#316]
Gradle config:
```kotlin
plugins {
    id("org.jmailen.kotlinter") version "3.13.0"
}

buildscript {
    configurations.classpath {
        resolutionStrategy {
            val ktlintVersion = "0.48.2"
            force(
                "com.pinterest.ktlint:ktlint-core:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-reporter-checkstyle:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-reporter-json:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-reporter-html:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-reporter-plain:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-reporter-sarif:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-ruleset-experimental:$ktlintVersion",
                "com.pinterest.ktlint:ktlint-ruleset-standard:$ktlintVersion"
            )
        }
    }
}
```

```
> Task :formatKotlinMain FAILED
1 actionable task: 1 executed
format worker execution error
java.lang.NoSuchMethodError: 'void com.pinterest.ktlint.core.KtLint$ExperimentalParams.<init>(java.lang.String, java.lang.String, java.lang.Iterable, java.util.Set, java.util.Map, kotlin.jvm.functions.Function2, boolean, java.lang.String, boolean, com.pinterest.ktlint.core.api.EditorConfigDefaults, com.pinterest.ktlint.core.api.EditorConfigOverride, boolean, int, kotlin.jvm.internal.DefaultConstructorMarker)'
	at org.jmailen.gradle.kotlinter.tasks.format.FormatWorkerAction.format(FormatWorkerAction.kt:85)
	at org.jmailen.gradle.kotlinter.tasks.format.FormatWorkerAction.formatKt(FormatWorkerAction.kt:78)
	at org.jmailen.gradle.kotlinter.tasks.format.FormatWorkerAction.access$formatKt(FormatWorkerAction.kt:20)
	at org.jmailen.gradle.kotlinter.tasks.format.FormatWorkerAction$execute$1$1.invoke(FormatWorkerAction.kt:44)
	at org.jmailen.gradle.kotlinter.tasks.format.FormatWorkerAction$execute$1$1.invoke(FormatWorkerAction.kt:20)
	at org.jmailen.gradle.kotlinter.tasks.format.FormatWorkerAction.execute(FormatWorkerAction.kt:51)
	at org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:63)
	at org.gradle.workers.internal.NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:66)
	at org.gradle.workers.internal.NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:62)
	at org.gradle.internal.classloader.ClassLoaderUtils.executeInClassloader(ClassLoaderUtils.java:100)
	at org.gradle.workers.internal.NoIsolationWorkerFactory$1.lambda$execute$0(NoIsolationWorkerFactory.java:62)
	at org.gradle.workers.internal.AbstractWorker$1.call(AbstractWorker.java:44)
	at org.gradle.workers.internal.AbstractWorker$1.call(AbstractWorker.java:41)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:199)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)
	at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:157)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)
	at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:73)
	at org.gradle.workers.internal.AbstractWorker.executeWrappedInBuildOperation(AbstractWorker.java:41)
	at org.gradle.workers.internal.NoIsolationWorkerFactory$1.execute(NoIsolationWorkerFactory.java:59)
	at org.gradle.workers.internal.DefaultWorkerExecutor.lambda$submitWork$2(DefaultWorkerExecutor.java:205)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.runExecution(DefaultConditionalExecutionQueue.java:187)
	at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.access$700(DefaultConditionalExecutionQueue.java:120)
	at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner$1.run(DefaultConditionalExecutionQueue.java:162)
	at org.gradle.internal.Factories$1.create(Factories.java:31)
	at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:249)
	at org.gradle.internal.work.DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:109)
	at org.gradle.internal.work.DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:114)
	at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.runBatch(DefaultConditionalExecutionQueue.java:157)
	at org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.run(DefaultConditionalExecutionQueue.java:126)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

FAILURE: Build failed with an exception.
```

**Comment by jeremymailen** [comment]
We're not upgraded yet to 0.48.2 and it requires API changes.
There are a few pending PRs such as https://github.com/jeremymailen/kotlinter-gradle/pull/279 that we were waiting on, but maybe it makes sense to push a minor release to support 0.48.2 sooner.

**Comment by kkocel** [comment]
It's very frustrating b/c the command line tool formats code differently than kotlinter...

**Comment by PaulWoitaschek** [comment]
It would be super great if this came up to date with ktlint. We are currently evaluating if we should switch to a lfs backed cli implementation.

**Comment by jeremymailen** [comment]
Ok, 3.14.0 is now released which brings us up to ktlint 0.48.2.
Thanks for everyone's patience.
